modules/eks/iam.tf

```diff
@@ -1,5 +1,36 @@
+resource "aws_iam_role" "eks_fargate_role" {
+  name = "EKSFargateRole"
+  force_detach_policies = true
+  assume_role_policy = data.aws_iam_policy_document.fargate_role_assume_document.json
+  
+  
+/* <<POLICY
+{
+    "Version": "2012-10-17",
+    "Statement": [{
+        "Effect": "Allow",
+        "Principal": {
+            "Service": ["eks.amazonaws.com", "eks-fargate-pods.amazonaws.com"]
+        },
+        "Action": "sts:AssumeRole"
+    }]
+}
+POLICY */
+
+}
+
+data "aws_iam_policy_document" "fargate_role_assume_document" {
+  statement {
+    actions = ["sts:AssumeRole"]
+    principals {
+      type = "Service"
+      identifiers = ["eks.amazonaws.com", "eks-fargate-pods.amazonaws.com"]
+    }
+  }
+}
+
 resource "aws_iam_role" "kube_rds_controller_role" {
-    name = "StupidRole"
+    name = "KubeRDSAccessRole"
```

Comment:
I believe it wasn't clear statement, but recently we decided to get rid of RDS controller. 
I assume we don't need this role anymore. 
@bertvi Could you share your opinion about it?